lib/skate_web/router.ex

```diff
@@ -29,6 +29,7 @@ defmodule SkateWeb.Router do
 
   pipeline :accepts_html do
     plug :accepts, ["html"]
+    plug :put_secure_browser_headers, %{"content-security-policy" => "default-src 'self'"}
```

Comment:
question: have you checked on a dev env if all our vendored code and external services (drift, appcues, fullstory, etc) if everything still works with this security policy?

_If_ I recall correctly, I think we may have some inline code executing from some of our libraries and this CSP may block that.

Reply:
btw, I think fullstory is only present on dev-blue

Reply:
I have not. I'm kind of tempted to roll at least part of this change back in order to keep the addition of Sobelow as lite as possible, and then add some follow-up tickets to investigate the specific issues it found, including the secure headers that I tried to fix here.

Reply:
Yeah... it don't work :(